functions/activate-batch-auto-background.js

```diff
@@ -142,6 +174,12 @@ async function handle(_, db, contract) {
             prizes: '[]'
           }
         await db.put('batches', batchItem)
+
+        const accountUpdateBadluckCount = {
+          ...addressBadluckCount[a]
```

Comment:
It is because in here I'm using that dictionary to retrieve the  #account with updated badluckcount, to update into db

Reply:
Ok maybe rename the variables to `existingAccounts` or something similar